### PR TITLE
Update the baseURL; add the deploy command and README of the website directory

### PR DESCRIPTION
**Pull request by oscarpolanco**
Update the baseURL; add the deploy command and README of the website directory.

# Steps for testing the site locally:
- Go to the `website` folder
- Install all dependencies using `npm install`
- Start your local server using `npm run start`
- Test locally at [localhost](http://localhost:3000/)

# Steps to test the build:
- Go to the `website` folder
- Install all dependencies using `npm install`
- Start your local server using `npm run build`
- Check that now you got a `build` folder under the `website` directory

### CODE REVIEW

**Review by jonwinton — APPROVED** [review]

File: website/README.md [hunk 1]
```diff
@@ -1,27 +1,26 @@
-This website was created with [Docusaurus](https://docusaurus.io/).
+You will need to to have `Node >= 8.x` and `Yarn >= 1.5`.
```
Comment (manuelurenah) [inline website/README.md]: Doubled `to` in here
```suggestion
You will need to have `Node >= 8.x` and `Yarn >= 1.5`.
```

File: website/README.md [hunk 2]
```diff
@@ -30,164 +29,39 @@ Your project file structure should look something like this
...
     static/
       css/
       img/
+    versioned_docs/
+    versioned_sidebars/
+    README.md
     package.json
     sidebar.json
     siteConfig.js
+    versions.json
```
Comment (manuelurenah) [inline website/README.md]: There's a bash utility called `tree` that helps to make beautiful project structures. You can install it using `brew install tree` and then generate the file structure. This is just being picky but I think is a nice addition to the docs

File: website/README.md (hunk 1 above)
Comment (oscarpolanco) [reply website/README.md]: Done!!! Thanks @manuelurenah

File: website/README.md (hunk 2 above)
Comment (oscarpolanco) [reply website/README.md]: Done!!!

**Review by manuelurenah — APPROVED** [review]

**Pull request merged**